pkgs/top-level/python-packages.nix

```diff
@@ -11224,7 +11224,7 @@ self: super: with self; {
 
   pydmd = callPackage ../development/python-modules/pydmd { };
 
-  pydns = callPackage ../development/python-modules/py3dns { };
+  py3dns = callPackage ../development/python-modules/py3dns { };
```

Comment:
Add an alias for the old name to python-aliases.nix

Reply:
done to the best of my ability.